src/publicHolidayReminder.js

```diff
@@ -0,0 +1,74 @@
+const { DateTime } = require('luxon');
+const ical = require('node-ical');
+
+const NZ_CALENDAR_URL =
+  'https://calendar.google.com/calendar/ical/en.new_zealand%23holiday%40group.v.calendar.google.com/public/basic.ics';
+
+// We have two options for the AU calendar. We are trying the Victoria gov one
+// first because it will (presumably) have more accurate data for Melbourne and
+// will avoid us having to filter out holidays from other states.
+// const AU_CALENDAR_URL = 'https://calendar.google.com/calendar/ical/en.australian.official%23holiday%40group.v.calendar.google.com/public/basic.ics'
+const AU_CALENDAR_URL =
+  'https://www.vic.gov.au/sites/default/files/2022-09/Victorian-public-holiday-dates.ics';
+
+const findRelevantEvents = (rawEvents, date) => {
+  return Object.values(rawEvents)
+    .filter(rawEvent => rawEvent.start?.toString() === date.toString())
+    .map(rawEvent => {
+      return {
+        start: rawEvent.start,
+        description: `${rawEvent.description} (${rawEvent.summary})`
+      };
+    });
+};
+
+const getCalendarData = async filePathOrUrl => {
+  let events = [];
+
+  if (filePathOrUrl.startsWith('http')) {
+    events = await ical.async.fromURL(filePathOrUrl);
+  } else {
+    events = await ical.async.parseFile(filePathOrUrl);
+  }
+
+  return events;
+};
```

Comment:
you might as well just return these directly:

```suggestion
const getCalendarData = async filePathOrUrl => {
  if (filePathOrUrl.startsWith('http')) {
    return ical.async.fromURL(filePathOrUrl);
  }

  return ical.async.parseFile(filePathOrUrl);
};
```